OpenCL.pc.in

```diff
@@ -0,0 +1,8 @@
+libdir=@CMAKE_INSTALL_FULL_LIBDIR@
+includedir=@CMAKE_INSTALL_FULL_INCLUDEDIR@
```

Comment:
Proposed change to allow easier relocation:
``` diff
-libdir=@CMAKE_INSTALL_FULL_LIBDIR@
-includedir=@CMAKE_INSTALL_FULL_INCLUDEDIR@
+prefix=@CMAKE_INSTALL_PREFIX@
+exec_prefix=${prefix}
+libdir=${exec_prefix}/lib
+includedir=${prefix}/include
```

Reply:
Thanks for the suggestion. If we decide to keep a single .pc file, rather than split the headers, I will most probably merge this. If we split the .pc files, then only `libdir` is used, and in this case I'd rather use the full version.

Reply:
Ok. The most important change wrt to relocation would be to keep `prefix` separate. Whether or not to include `includedir` doesn't matter for that question.

Reply:
Do you want to do a formal suggestion so that I can merge it? I will merge it right away.

Reply:
I don't know how to do that. Is that a PR to your branch?
If so: https://github.com/Kerilk/OpenCL-ICD-Loader/pull/1

Reply:
Thanks for the contribution, merged below.